ksqldb-common/src/main/java/io/confluent/ksql/metrics/ConsumerCollector.java

```diff
@@ -105,6 +113,8 @@ private void record(
     topicSensors.computeIfAbsent(getCounterKey(topic), k ->
         new TopicSensors<>(topic, buildSensors(k))
     ).increment(record, isError);
+    totalBytesSum.record(record == null ? 0.0
```

Comment:
Curious why not just record as a long instead of a double?

Reply:
The KsqlEngineMetrics, totalBytesIn metric and the existing consumer total bytes metric were all Double so I just used that type also for this new metric